frontend/app/templates/projects/create-new.hbs

```diff
@@ -0,0 +1,143 @@
+{{page-title 'Track New'}}
+{{#each @model.dependencyVersions as |dv|}}
+  {{dv.dependency.name}}
+  {{dv.version}}
+{{/each}}
+{{! TODO: remove this }}
```

Comment:
Do that :wink:

Reply:
done in https://github.com/adfinis/Outdated/pull/14/commits/6db7a0802e07c2c9fa721986935dbd350fcb5e4e :wink: